### DISCUSSION TITLE
GNOME menu entry is gone for some reason

### DISCUSSION THREAD

**Discussion by jarmos-san** [kovidgoyal/kitty#4640] [Q&A] [answered]
Kitty is working as fine as expected but it's menu entry is gone after a quick restart of the system. I'm using GNOME for more context & a `kitty.desktop` file does exist at: `~/.local/share/applications/kitty.desktop`. Also here are the contents of the `kitty.desktop` file:

```desktop
[Desktop Entry]
Version=1.0
Type=Application
Name=kitty
GenericName=Terminal emulator
Comment=Fast, feature-rich, GPU based terminal
TryExec=kitty
Exec=kitty
Icon=/home/space/.local/kitty.app/share/icons/hicolor/256x256/apps/kitty.png
Categories=System;TerminalEmulator;
```

Is there something wrong from my end & how do I fix this issue?

**Comment by page-down** [comment, marked as answer]
> ... but it's menu entry is gone ...

My guess is that the kitty is not found by TryExec.
Maybe you can try the following configuration, does it work?

```ini
[Desktop Entry]
TryExec=/abs/path/to/kitty
```

**Reply by jarmos-san** [reply]
Hmm...interesting, the following change fixed the issue for whatever reason though. :thinking: 

```desktop
...
Exec=/abs/path/to/kitty
...
```
What could've gone wrong?

**Reply by page-down** [reply]
The (GUI) shell cannot find kitty in the PATH, you can try to add the kitty bin folder to the system-level PATH. Or create a symbolic link to put kitty into the PATH.

**Reply by jarmos-san** [reply]
Hmm...I'm pretty sure `kitty` is available on `PATH`, yet the desktop entry can't find it.
